packages/react-web/package.json

```diff
@@ -74,6 +75,7 @@
   },
   "peerDependencies": {
     "@lens-protocol/metadata": "^1.0.0",
+    "@xmtp/react-sdk": "^3.0.0",
```

Comment:
Are we in a position to make this also optional?

Reply:
they are, see below

```
"peerDependenciesMeta": {
    "@xmtp/react-sdk": {
      "optional": true
    }
  },
```

Reply:
can you please double check the import path? basically only importing the relevant hook should end up importing the XMTP SDK

Reply:
yes, I checked, all xmtp imports are in the inbox folder, which is even exported as `@lens-protocol/react-web/inbox`. I also tested example web app without xmtp peer dep and of course without inbox hooks, it all works fine.